content/guides/webpack-and-typescript.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Webpack & Typescript
+sort: 20
+contributors:
+  - morsdyce
+---
+
+Typescript is a typed superset of javascript that compiles to plain javascript, in this guide we will learn how to integrate Typescript with webpack.
+To learn more about Typescript [click here](https://www.typescriptlang.org).
```

Comment:
You can replace this sentence by linking to TypeScript in the previous paragraph.